screens/Navigation/ConversationRequestsListNav.tsx

```diff
@@ -151,35 +159,32 @@ export default function ConversationRequestsListNav() {
     <NativeStack.Screen name="ChatsRequests" options={navigationOptions}>
       {(navigationProps) => {
         navRef.current = navigationProps.navigation;
+        let items: ConversationFlatListItem[] = likelyNotSpam;
+        if (likelySpam.length > 0) {
+          items = isSpamToggleEnabled
+            ? [
+                ...likelyNotSpam,
+                {
+                  topic: "hiddenRequestsButton",
```

Comment:
Is this how we render non "list item" components within the list itself?

Reply:
It's a list item, just not a conversation ! But yeah it's a way to render a different type of item in the conversation flash list. We use it only for that specific android toggle but could be useful for other stuff in the future